equinix/resource_network_device.go

```diff
@@ -1062,6 +1080,12 @@ func createNetworkDevices(d *schema.ResourceData) (*ne.Device, *ne.Device) {
 	if v, ok := d.GetOk(neDeviceSchemaNames["ProjectID"]); ok {
 		primary.ProjectID = ne.String(v.(string))
 	}
+	if v, ok := d.GetOk(neDeviceSchemaNames["DiverseFromDeviceUUID"]); ok {
+		primary.DiverseFromDeviceUUID = ne.String(v.(string))
+	}
+	if v, ok := d.GetOk(neDeviceSchemaNames["DiverseFromDeviceName"]); ok {
```

Comment:
This should be removed as we are not sending diverse device name in the request.

Reply:
removed